authorization/src/main/kotlin/com/ritense/authorization/permission/Permission.kt

```diff
@@ -141,14 +170,14 @@ data class Permission(
                         root,
                         customQuery,
                         criteriaBuilder,
-                        resourceType,
+                        request.resourceType,
                         queryDialectHelper
                     )
                 }.toTypedArray()
             )
     }
 
-    private fun <U> appliesInContext(
+    fun <U> appliesInContext(
```

Comment:
This method is short, but hard to understand. Can you break the boolean checks up?

Reply:
I don't see how it is hard to understand, it's an if statement with parenthesis. If you want to make it more readable, I'm assuming you have something in mind. Can you make the change yourself?

Reply:
done